Platform/Advantech/RSB4411_iMX6Q_1GB/AcpiTables/Dsdt-Uart.asl

```diff
@@ -72,6 +72,13 @@ Device (UAR1)
       ,
     )
   })
+
+  Name(_DSD, Package() {

```

Comment:
I think EDK2 source formatting rules require a space between function name and opening parenthesis.

Reply:
Yes, you're right.